test/features/stepDefinitions/welcome.steps.js

```diff
@@ -2,11 +2,11 @@ const {Given, When, Then} = require("@cucumber/cucumber");
 const Pages = require('../../src/pages');
 
 Given(/^I launch the app$/, async function () {
-    await Pages.welcomePage.launchApp();
+    await Pages.basePage.launchApp();
```

Comment:
this is not needed anymore, we can call directly the ActionsHelper. launchApp(), already in master

Reply:
I will do it